common/test/acceptance/tests/helpers.py

```diff
@@ -267,7 +270,7 @@ def generate_course_key(org, number, run):
     return CourseLocator(org, number, run, deprecated=(default_store == 'draft'))
 
 
-def select_option_by_value(browser_query, value):
+def select_option_by_value(browser_query, value, focus_out=False):
```

Comment:
Not a problem, I can change the default to True. The case where this should be False is for use with a select input in `header.html` that forces a form submission onChange for language preference that is set in `lang_edx.js`. This element is used in the test_lms_language_change test and the test_studio_language_change tests where it awaits the redirect after the form submission to continue the test.